src/main/java/io/neonbee/handler/ErrorHandler.java

```diff
@@ -0,0 +1,21 @@
+package io.neonbee.handler;
+
+import io.neonbee.NeonBee;
+import io.vertx.core.Future;
+
+/**
+ * The class which implements this interface MUST have a default constructor. Otherwise NeonBee can't instantiate the
+ * class during bootstrap phase.
+ */
+public interface ErrorHandler extends io.vertx.ext.web.handler.ErrorHandler {
+
+    /**
+     * The purpose of this method is to offer the possibility to also executed blocking code during the initialization
+     * of the ErrorHandler. This is beneficial in case that the error template needs to be read from a database or file.
+     *
+     * @param neonbee           The related NeonBee instance
+     * @param errorTemplateName The name of the error template
+     * @return a Future with a reference of this ErrorHandler
+     */
+    Future<ErrorHandler> initialize(NeonBee neonbee, String errorTemplateName);
```

Comment:
Better to use Future<Void> here.
Also should we rather use Vertx vertx, instead of NeonBee? And is it NeonBee neonBee, oder neonBee?
Also one additional thing came to my mind:
The errorTemplateName could stay part of the constructor, while the initialize then is reduced to simply:
Future<Void> initialize(Vertx vertx) or Future<Void> initialize(NeonBee neonBee)

Reply:
If I want to write y custom handler that wants to benefit from a errorTemplateName, how du I know that this is possible? Of course I can read the JavaDoc and maybe don't miss the point that I'm able to do this. But if it is part of the signature I have to implement, it is more visible to me.

Reply:
I passed Vert.x in the first version, but then I thought: "what if I need a config from the config directory to initialize the handler?" Of course I can do NeonBee.get(vertx), but I can also do neonBee.gertVertx() :)

I don't have strong feelings here, I will do what you prefer :)

Reply:
We agreed on passing only NeonBee and add a getServerConfig() method to NeonBee. errorTemplateName can then be retrieved via the ServerConfig.

ServerConfig will be stored in the NeonBee context